### PR TITLE
Add UI Changes to Grouping

### PR DESCRIPTION
**Pull request by joeizang**
based on discussions and meeting that resulted in the following issues: #42

### CODE REVIEW

File: packages/react-app/src/components/Account.tsx [hunk 1]
```diff
@@ -206,18 +214,18 @@ export default function Account({ minimized }) {
       displayToast()
       return
     }
-    window.ethereum.on('chainChanged', async chainId => {
-      const accounts = await window.ethereum.request({
+    (window.ethereum as any).on('chainChanged', async (chainId: string | number | any) => {
```
Comment (yann300) [inline packages/react-app/src/components/Account.tsx:217]: see https://docs.metamask.io/guide/ethereum-provider.html#chainchanged
this is a string

File: packages/react-app/src/components/BadgesPaginatedSection.tsx [hunk 2]
```diff
@@ -92,43 +93,43 @@ export default function BadgesPaginatedSection({
               <AddressedCard badges={badges} etherscanRef={etherscanRef} />
             </Grid>
           ) : groupedRewards && groupedRewards.length > 0 ? (
-            groupedRewards.map(event => {
+            groupedRewards.map((eventB: any) => {
```
Comment (yann300) [inline packages/react-app/src/components/BadgesPaginatedSection.tsx:96]: can you type the `event` object and also the the return of `tokensData`? that will help see more clearly what's going on.

File: packages/react-app/src/components/NetInfo.tsx [hunk 3]
```diff
@@ -1,15 +1,16 @@
...
-export function NetInfo({ netInfo, setNetInfo, connectedAddress, checkForWeb3Provider, displayToast }) {
+export function NetInfo({ netInfo, setNetInfo, connectedAddress, checkForWeb3Provider, displayToast }: NetInfoProps) {
   useEffect(() => {
     if (checkForWeb3Provider() === 'Not Found') {
       console.log('Metamask is not installed!')
       // displayToast()
       return
     }
-    window.ethereum.on('chainChanged', async chainId => {
+    (window as any).ethereum.on('chainChanged', async (chainId: string | number | any) => {
```
Comment (yann300) [inline packages/react-app/src/components/NetInfo.tsx:13]: same, chainId is a string

**Review by yann300 — APPROVED** [review]

File: packages/react-app/src/components/Account.tsx (hunk 1 above)
Comment (joeizang) [reply packages/react-app/src/components/Account.tsx:217]: Thanks for pointing this out. I definitely wasn't sure of the type. I will ammend this everywhere.

File: packages/react-app/src/components/Account.tsx [hunk 4]
```diff
@@ -206,18 +214,18 @@ export default function Account({ minimized }) {
...
+      const accounts: any[] = await (window.ethereum as any).request({
```
[the remark this answers is not in the document]
Comment (joeizang) [reply packages/react-app/src/components/Account.tsx:218]: You are right

**Pull request merged**